src/Car.cpp

```diff
@@ -0,0 +1,40 @@
+#include <iostream>
+
+#include "Car.hpp"
+
+constexpr int topSpeed = 200;
+
+Car::~Car() {
+    std::cout << __FUNCTION__ << std::endl;
+}
+void Car::turnLeft() {
+    std::cout << __FUNCTION__ << std::endl;
+}
+void Car::turnRight() {
+    std::cout << __FUNCTION__ << std::endl;
+}
+void Car::brake(int power) {
+    if(speed_ > power){
+        speed_ -= power;
+    }
+    else{
+        speed_ = 0;
+    }
+    
+}
+void Car::accelerate(int speed) {
+    std::cout << __FUNCTION__ << std::endl;
+
+    auto isValidAccelerate = [](const auto& speed) { return speed >= -15 && speed <= 100; };
```

Comment:
Mayble lets make values -15 and 100 as const or constexpr varaibles?

Reply:
+1